ZigbeeNet/CC/CCZnp.cs

```diff
@@ -260,40 +166,36 @@ private void Unpi_DataReceived(object sender, SerialPacket e)
                                 current.Response(result);
                             }
                         };
-                        _sreqRunning.Parse((MessageType)e.Type, e.Length, e.Payload);
+                        _sreqRunning.Parse(e.Type, e.Length, e.Payload);
                     }
-                }
             }
         }
 
         public void Start()
         {
             Enabled = true;
-            Unpi.Open();
+            unpi.Open();
         }
 
         public void Stop()
         {
             Enabled = false;
             _requestQueue = new ConcurrentQueue<ZpiObject>();
-            Unpi = null;
+            unpi = null;
 
             Closed?.Invoke(this, EventArgs.Empty);
         }
 
         public byte[] SendCommand(MessageType type, SubSystem subSystem, byte commandId, byte[] payload)
         {
-            return Unpi.Send((int)type, (int)subSystem, commandId, payload);
+            return unpi.Send((int) type, (int) subSystem, commandId, payload);
         }
 
         public void Request(SubSystem subSystem, byte cmdId, ArgumentCollection reqestArgs)
         {
-            if (Unpi == null)
-            {
-                throw new NullReferenceException("CCZnp has not been initialized yet");
-            }
+            if (unpi == null) throw new NullReferenceException("CCZnp has not been initialized yet");
```

Comment:
We should avoid throwing NullReferenceException manually like mentioned by @nicolaiw in Issue #15

Reply:
Not my code.

Reply:
Oh i thought i fixed it already. Sorry